pkg/operator/operator.go

```diff
@@ -147,13 +147,45 @@ func (optr *Operator) Run(workers int, stopCh <-chan struct{}) {
 		return
 	}
 	glog.Info("Synched up caches")
+	go wait.Poll(machineRolloutPollInterval, machineRolloutTimeout, func() (bool, error) {
+		return deployMachines(*optr)
+	})
+
 	for i := 0; i < workers; i++ {
 		go wait.Until(optr.worker, time.Second, stopCh)
 	}
 
 	<-stopCh
 }
 
+func deployMachines(optr Operator) (bool, error) {
+	operatorConfig, err := render.Config(optr.config)
+	if err != nil {
+		glog.Fatalf("Error decoding operator config: %v", err)
```

Comment:
This still won't return. Is that the intention? Why can't we drop out of the calling function with an error?